scripts/check_committers.py

```diff
@@ -64,18 +74,24 @@ def get_local_roster():
         elif role == 'Committer':
             committers.add(uid)
         else:
-            raise ValueError(f"Invalid role {role!r} for {uid}")
+             raise ValueError(f"Invalid role {role!r} for {uid}")
     return Roster(committers, pmcs)
 
-
 if __name__ == "__main__":
+    with open(committers_yaml, "r") as f:
+        d = yaml.safe_load(f)
```

Comment:
It doesn't seem this variable is used, is it?

Reply:
Thanks. Removed.